### ISSUE TITLE
deferred error : (error "Got error 403: (error http 403)")

### ISSUE THREAD

**Issue by hrqmonteiro** [kidd/org-gcal.el#247]
![Screenshot 2024-04-16 at 00 34 58](https://github.com/kidd/org-gcal.el/assets/17055027/ebc27efe-cbbb-472c-b612-9d5b9c3beaf4)

I have that when i do `org-gcal-fetch`

**Comment by telotortium** [comment]
That probably means you have not logged into your Google account or the token has expired. Have you tried `org-gcal-refresh`?

**Comment by hrqmonteiro** [comment]
> That probably means you have not logged into your Google account or the token has expired. Have you tried `org-gcal-refresh`?

This command doesn't exist.

![Screenshot 2024-04-18 at 08 42 05](https://github.com/kidd/org-gcal.el/assets/17055027/7eac41bf-4451-4713-93fc-5290ae75c478)

**Comment by hrqmonteiro** [comment]
> [@hrqmonteiro](https://github.com/hrqmonteiro) did you get it to work?

No, no one offered any help so i had to give up. The package is basically useless.
